manifest.webapp

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Passwords",
+  "slug": "passwords",
+  "icon": "icon.svg",
+  "categories": [],
+  "version": "0.1.0",
+  "licence": "AGPL-3.0",
+  "editor": "",
```

Comment:
Is it normal that `editor` is empty?

Reply:
Nope, thanks https://github.com/cozy/cozy-passwords/pull/1/commits/f2d2498dbeafd462f762c5bc9b431a146d9e76b4